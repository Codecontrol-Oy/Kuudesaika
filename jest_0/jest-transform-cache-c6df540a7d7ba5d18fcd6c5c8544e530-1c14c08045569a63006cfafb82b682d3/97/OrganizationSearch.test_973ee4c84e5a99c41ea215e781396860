'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _OrganizationSearch = require('./OrganizationSearch');var _OrganizationSearch2 = _interopRequireDefault(_OrganizationSearch);
var _Organizations = require('../../../mock/Organizations.json');var _Organizations2 = _interopRequireDefault(_Organizations);
var _Article = require('grommet/components/Article');var _Article2 = _interopRequireDefault(_Article);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

describe('<OrganizationSearch />', function () {

    it('Should match snapshot', function () {
        var organizationSearch = mount(
        _react2.default.createElement(_OrganizationSearch2.default, { organizations: _Organizations2.default.organizations }));

        expect(organizationSearch).toMatchSnapshot();
    });

    it('Should render search results', function () {
        var organizationSearch = mount(
        _react2.default.createElement(_OrganizationSearch2.default, { organizations: _Organizations2.default.organizations }));

        expect(organizationSearch.find('OrganizationSearchResult').length).toEqual(0);
        expect(organizationSearch.find('input').length).toEqual(1);
        var event = {
            "target": {
                "value": "valtuusto" } };


        organizationSearch.instance().onSearch(event);
        expect(organizationSearch.find('OrganizationSearchResult').length).not.toEqual(0);

    });

    it('Should render no search results', function () {
        var organizationSearch = mount(
        _react2.default.createElement(_OrganizationSearch2.default, { organizations: _Organizations2.default.organizations }));

        expect(organizationSearch.find('OrganizationSearchResult').length).toEqual(0);
        expect(organizationSearch.find('input').length).toEqual(1);
        var event = {
            "target": {
                "value": "ABCDEFG" } };


        organizationSearch.instance().onSearch(event);
        expect(organizationSearch.find('OrganizationSearchResult').length).toEqual(0);
        expect(organizationSearch.find('article').length).toEqual(1);
    });
});