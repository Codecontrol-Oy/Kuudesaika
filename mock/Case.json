{
    "function": {
            "data" : {
                "url": "https://paatos-helsinki.6aika.fi/v1/function/1/",
                "id": 1,
                "data_source": "open_ahjo",
                "created_at": "2017-05-05T06:13:06.497373Z",
                "modified_at": "2017-05-05T06:13:06.497436Z",
                "origin_id": "1",
                "name": "Hallintoasiat",
                "function_id": "00",
                "parent": null               
            }
    },
    "cases": [
        {
            "url": "https://paatos-helsinki.6aika.fi/v1/case/1/",
            "id": 1,
            "data_source": "open_ahjo",
            "actions": [
                "https://paatos-helsinki.6aika.fi/v1/action/20/"
            ],
            "geometries": [],
            "created_at": "2017-05-05T06:14:45.974122Z",
            "modified_at": "2017-05-05T06:14:45.974158Z",
            "origin_id": "7",
            "title": "Asuntolautakunnan esittelysuunnitelma kevätkaudelle 2012",
            "register_id": "HEL 2012-002954",
            "function": "https://paatos-helsinki.6aika.fi/v1/function/9/",
            "attachments": []
        },
        {
            "url": "https://paatos-helsinki.6aika.fi/v1/case/2/",
            "id": 2,
            "data_source": "open_ahjo",
            "actions": [
                "https://paatos-helsinki.6aika.fi/v1/action/9293/"
            ],
            "geometries": [],
            "created_at": "2017-05-05T06:14:45.986709Z",
            "modified_at": "2017-05-05T06:14:45.986746Z",
            "origin_id": "3995",
            "title": "Staran kaupunkitekniikoiden läntisen alueen pienet jyräpäällystystyöt 2013",
            "register_id": "HEL 2012-016286",
            "function": "https://paatos-helsinki.6aika.fi/v1/function/194/",
            "attachments": []
        },
        {
            "url": "https://paatos-helsinki.6aika.fi/v1/case/3/",
            "id": 3,
            "data_source": "open_ahjo",
            "actions": [
                "https://paatos-helsinki.6aika.fi/v1/action/39/"
            ],
            "geometries": [],
            "created_at": "2017-05-05T06:14:45.997440Z",
            "modified_at": "2017-05-05T06:14:45.997473Z",
            "origin_id": "16",
            "title": "Asukkaiden valinta vapautuviin ARA-asuntoihin",
            "register_id": "HEL 2012-005707",
            "function": "https://paatos-helsinki.6aika.fi/v1/function/450/",
            "attachments": []
        }]
}