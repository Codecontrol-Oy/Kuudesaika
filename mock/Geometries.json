{
     "geometries": [
        {
            "data_source": "open_ahjo",
            "created_at": "2017-05-05T06:13:41.855615Z",
            "modified_at": "2017-05-05T06:13:41.855672Z",
            "origin_id": "5",
            "name": "Alakiventie 5",
            "type": "address",
            "geometry": {
                "type": "Point",
                "coordinates": [
                    25.077758683528398,
                    60.22071379562326
                ]
            }
        },
        {
            "data_source": "open_ahjo",
            "created_at": "2017-05-05T06:13:47.412236Z",
            "modified_at": "2017-05-05T06:13:47.412271Z",
            "origin_id": "827",
            "name": "91-45-145-5",
            "type": "plan_unit",
            "geometry": {
                "type": "Polygon",
                "coordinates": [
                    [
                        [
                            25.078492370173475,
                            60.22155576522049
                        ],
                        [
                            25.07898612923674,
                            60.2209582329813
                        ],
                        [
                            25.07899939996638,
                            60.2209399601969
                        ],
                        [
                            25.07900944001587,
                            60.22092117773851
                        ],
                        [
                            25.079016186612012,
                            60.220902038225596
                        ],
                        [
                            25.079019567918284,
                            60.22088267633203
                        ],
                        [
                            25.079019575286875,
                            60.22086324464457
                        ],
                        [
                            25.079016200048645,
                            60.22084388677433
                        ],
                        [
                            25.079009478637996,
                            60.22082474630537
                        ],
                        [
                            25.07899944748894,
                            60.22080596682146
                        ],
                        [
                            25.07898617909662,
                            60.22078768290929
                        ],
                        [
                            25.07896979110206,
                            60.22077004707923
                        ],
                        [
                            25.07895040105921,
                            60.22075317594003
                        ],
                        [
                            25.07892813560655,
                            60.22073721302105
                        ],
                        [
                            25.07779082358496,
                            60.219992264434524
                        ],
                        [
                            25.07779082358496,
                            60.219992264434524
                        ],
                        [
                            25.07778102238484,
                            60.219986382334596
                        ],
                        [
                            25.077770247993417,
                            60.21998093162229
                        ],
                        [
                            25.077758581722932,
                            60.21997596610199
                        ],
                        [
                            25.077746104800237,
                            60.219971503676604
                        ],
                        [
                            25.077732907536436,
                            60.21996758916984
                        ],
                        [
                            25.077719098219642,
                            60.21996424046887
                        ],
                        [
                            25.077704758118717,
                            60.21996149342736
                        ],
                        [
                            25.077690004542095,
                            60.2199593659272
                        ],
                        [
                            25.077674945756275,
                            60.21995786688031
                        ],
                        [
                            25.077659681028432,
                            60.219957014179165
                        ],
                        [
                            25.07680422347679,
                            60.21992742663691
                        ],
                        [
                            25.076651548121873,
                            60.22101977300276
                        ],
                        [
                            25.07779196202046,
                            60.22153154925718
                        ],
                        [
                            25.078492370173475,
                            60.22155576522049
                        ]
                    ]
                ]
            }
        }
    ]
}