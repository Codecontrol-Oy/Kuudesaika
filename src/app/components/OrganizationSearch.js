import React from 'react';
import Section from 'grommet/components/Section';
import Search from 'grommet/components/Search';
import List from 'grommet/components/List';
import Article from 'grommet/components/Article';
import OrganizationSearchResult from './OrganizationSearchResult.js';
import PropTypes from 'prop-types';

export default class OrganizationSearch extends React.Component {
  constructor (props) {
    super(props);
    this.state = {
        searchResults: null,
        searchStarted: false
    };
  }

  onSearch = (event) => {
    let Results = [];
    let searchStarted = false;
    if (event.target.value && event.target.value.length > 2) {
      searchStarted = true;
      for (let i = 0; i < this.props.organizations.length - 1; i++) {
          const resultsFound = this.props.organizations[i].name.toLowerCase().includes(event.target.value.toLowerCase());
          if (resultsFound) {
                  Results.push(<OrganizationSearchResult key={Results.length + 1} organization={this.props.organizations[i]} />);
          }
      }
    } else {
      searchStarted = false;
    }
    if (Results.length > 0) {
        this.setState({searchResults: <List>{Results}</List>, searchStarted});
    } else {
        this.setState({searchResults: null, searchStarted});
    }
  }
  render () {

    return (
            <Section>
                <Search onDOMChange={this.onSearch} placeHolder={"Hakusana"} responsive={false} inline size={"medium"} />
                {this.state.searchResults ? this.state.searchResults : this.state.searchStarted ? <Article>Ei tuloksia.</Article> : ''}
            </Section>

    );
  }
}

OrganizationSearch.propTypes = {
  organizations: PropTypes.array
};
