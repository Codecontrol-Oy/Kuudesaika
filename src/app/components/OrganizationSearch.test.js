import React from 'react';
import OrganizationSearch from './OrganizationSearch';
import mock from '../../../mock/Organizations.json';
import Article from 'grommet/components/Article';

describe('<OrganizationSearch />', () => {

    it('Should match snapshot', () => {
        const organizationSearch = mount(
            <OrganizationSearch organizations={mock.organizations}  />
        );
        expect(organizationSearch).toMatchSnapshot();
    });

    it('Should render search results', () => {
         const organizationSearch = mount(
            <OrganizationSearch organizations={mock.organizations}  />
        );
        expect(organizationSearch.find('OrganizationSearchResult').length).toEqual(0);
        expect(organizationSearch.find('input').length).toEqual(1);
        const event = {
            "target" : {
                "value" : "valtuusto"
            }
        };
        organizationSearch.instance().onSearch(event);
        expect(organizationSearch.find('OrganizationSearchResult').length).not.toEqual(0);

    });

    it('Should render no search results', () => {
         const organizationSearch = mount(
            <OrganizationSearch organizations={mock.organizations}  />
        );
        expect(organizationSearch.find('OrganizationSearchResult').length).toEqual(0);
        expect(organizationSearch.find('input').length).toEqual(1);
        const event = {
            "target" : {
                "value" : "ABCDEFG"
            }
        };
        organizationSearch.instance().onSearch(event);
        expect(organizationSearch.find('OrganizationSearchResult').length).toEqual(0);
        expect(organizationSearch.find('article').length).toEqual(1);
    });
});