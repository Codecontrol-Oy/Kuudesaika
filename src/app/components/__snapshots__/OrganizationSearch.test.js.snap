// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrganizationSearch /> Should match snapshot 1`] = `
<OrganizationSearch
  organizations={
    Array [
      Object {
        "classification": "Kaupunki",
        "created_at": "2017-05-05T06:12:49.768550Z",
        "data_source": "helsinki",
        "dissolution_date": null,
        "events": Array [],
        "founding_date": null,
        "id": 1,
        "modified_at": "2017-05-05T06:12:49.768610Z",
        "name": "Helsingin kaupunki",
        "origin_id": "00001",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/1/",
      },
      Object {
        "classification": "Virasto",
        "created_at": "2017-05-05T06:12:49.780760Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 2,
        "modified_at": "2017-05-05T06:12:49.780799Z",
        "name": "Hallintokeskus",
        "origin_id": "01101",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/2/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.788757Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 3,
        "modified_at": "2017-05-05T06:12:49.788791Z",
        "name": "Hallinto-osasto/Halke",
        "origin_id": "011010",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/3/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.798209Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 4,
        "modified_at": "2017-05-05T06:12:49.798244Z",
        "name": "Kansainvälinen toiminta",
        "origin_id": "011013",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/4/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.807400Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 5,
        "modified_at": "2017-05-05T06:12:49.807434Z",
        "name": "Viestintä",
        "origin_id": "011016",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/5/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.819356Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 6,
        "modified_at": "2017-05-05T06:12:49.819418Z",
        "name": "Oikeuspalvelut",
        "origin_id": "011030",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/6/",
      },
      Object {
        "classification": "Virasto",
        "created_at": "2017-05-05T06:12:49.828239Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 7,
        "modified_at": "2017-05-05T06:12:49.828294Z",
        "name": "Talous- ja suunnittelukeskus",
        "origin_id": "01500",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/7/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.839674Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 8,
        "modified_at": "2017-05-05T06:12:49.839712Z",
        "name": "Taloussuunnittelu- ja varainhallintaosasto",
        "origin_id": "015100",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/8/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.848798Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 9,
        "modified_at": "2017-05-05T06:12:49.848833Z",
        "name": "Kehittämisosasto",
        "origin_id": "015200",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/9/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.862629Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 10,
        "modified_at": "2017-05-05T06:12:49.862700Z",
        "name": "Sisäinen tarkastus",
        "origin_id": "015300",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/10/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.875552Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 11,
        "modified_at": "2017-05-05T06:12:49.875623Z",
        "name": "Tietotekniikkaosasto",
        "origin_id": "015400",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/11/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.889101Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 12,
        "modified_at": "2017-05-05T06:12:49.889168Z",
        "name": "Varainhallinta",
        "origin_id": "015500",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/12/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.902636Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 13,
        "modified_at": "2017-05-05T06:12:49.902703Z",
        "name": "Elinkeinopalvelu",
        "origin_id": "015600",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/13/",
      },
      Object {
        "classification": "Virasto",
        "created_at": "2017-05-05T06:12:49.921509Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 14,
        "modified_at": "2017-05-05T06:12:49.921550Z",
        "name": "Henkilöstökeskus",
        "origin_id": "01800",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/14/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.933902Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 15,
        "modified_at": "2017-05-05T06:12:49.933951Z",
        "name": "Henkilöstöpolitiikkaosasto",
        "origin_id": "018010",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/15/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.948763Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 16,
        "modified_at": "2017-05-05T06:12:49.948812Z",
        "name": "Henkilöstövoimavaraosasto",
        "origin_id": "018050",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/16/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.961590Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 17,
        "modified_at": "2017-05-05T06:12:49.961635Z",
        "name": "Maahanmuutto-osasto",
        "origin_id": "018060",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/17/",
      },
      Object {
        "classification": "Virasto",
        "created_at": "2017-05-05T06:12:49.973059Z",
        "data_source": "helsinki",
        "dissolution_date": "2015-12-31",
        "events": Array [],
        "founding_date": null,
        "id": 18,
        "modified_at": "2017-05-05T06:12:49.973116Z",
        "name": "Työterveyskeskus",
        "origin_id": "02000",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/18/",
      },
      Object {
        "classification": "Valtuusto",
        "created_at": "2017-05-05T06:12:49.984343Z",
        "data_source": "helsinki",
        "dissolution_date": null,
        "events": Array [
          "https://paatos-helsinki.6aika.fi/v1/event/135/",
          "https://paatos-helsinki.6aika.fi/v1/event/171/",
          "https://paatos-helsinki.6aika.fi/v1/event/172/",
          "https://paatos-helsinki.6aika.fi/v1/event/173/",
          "https://paatos-helsinki.6aika.fi/v1/event/174/",
          "https://paatos-helsinki.6aika.fi/v1/event/175/",
          "https://paatos-helsinki.6aika.fi/v1/event/176/",
          "https://paatos-helsinki.6aika.fi/v1/event/177/",
          "https://paatos-helsinki.6aika.fi/v1/event/178/",
          "https://paatos-helsinki.6aika.fi/v1/event/179/",
          "https://paatos-helsinki.6aika.fi/v1/event/180/",
          "https://paatos-helsinki.6aika.fi/v1/event/181/",
          "https://paatos-helsinki.6aika.fi/v1/event/182/",
          "https://paatos-helsinki.6aika.fi/v1/event/183/",
          "https://paatos-helsinki.6aika.fi/v1/event/184/",
          "https://paatos-helsinki.6aika.fi/v1/event/185/",
          "https://paatos-helsinki.6aika.fi/v1/event/186/",
          "https://paatos-helsinki.6aika.fi/v1/event/187/",
          "https://paatos-helsinki.6aika.fi/v1/event/188/",
          "https://paatos-helsinki.6aika.fi/v1/event/189/",
          "https://paatos-helsinki.6aika.fi/v1/event/190/",
          "https://paatos-helsinki.6aika.fi/v1/event/191/",
          "https://paatos-helsinki.6aika.fi/v1/event/192/",
          "https://paatos-helsinki.6aika.fi/v1/event/193/",
          "https://paatos-helsinki.6aika.fi/v1/event/194/",
          "https://paatos-helsinki.6aika.fi/v1/event/195/",
          "https://paatos-helsinki.6aika.fi/v1/event/564/",
          "https://paatos-helsinki.6aika.fi/v1/event/565/",
          "https://paatos-helsinki.6aika.fi/v1/event/566/",
          "https://paatos-helsinki.6aika.fi/v1/event/711/",
          "https://paatos-helsinki.6aika.fi/v1/event/712/",
          "https://paatos-helsinki.6aika.fi/v1/event/713/",
          "https://paatos-helsinki.6aika.fi/v1/event/714/",
          "https://paatos-helsinki.6aika.fi/v1/event/715/",
          "https://paatos-helsinki.6aika.fi/v1/event/716/",
          "https://paatos-helsinki.6aika.fi/v1/event/717/",
          "https://paatos-helsinki.6aika.fi/v1/event/845/",
          "https://paatos-helsinki.6aika.fi/v1/event/846/",
          "https://paatos-helsinki.6aika.fi/v1/event/847/",
          "https://paatos-helsinki.6aika.fi/v1/event/848/",
          "https://paatos-helsinki.6aika.fi/v1/event/849/",
          "https://paatos-helsinki.6aika.fi/v1/event/986/",
          "https://paatos-helsinki.6aika.fi/v1/event/987/",
          "https://paatos-helsinki.6aika.fi/v1/event/988/",
          "https://paatos-helsinki.6aika.fi/v1/event/989/",
          "https://paatos-helsinki.6aika.fi/v1/event/990/",
          "https://paatos-helsinki.6aika.fi/v1/event/991/",
          "https://paatos-helsinki.6aika.fi/v1/event/1110/",
          "https://paatos-helsinki.6aika.fi/v1/event/1125/",
          "https://paatos-helsinki.6aika.fi/v1/event/1126/",
          "https://paatos-helsinki.6aika.fi/v1/event/1127/",
          "https://paatos-helsinki.6aika.fi/v1/event/1196/",
          "https://paatos-helsinki.6aika.fi/v1/event/1204/",
          "https://paatos-helsinki.6aika.fi/v1/event/1231/",
          "https://paatos-helsinki.6aika.fi/v1/event/1254/",
          "https://paatos-helsinki.6aika.fi/v1/event/1271/",
          "https://paatos-helsinki.6aika.fi/v1/event/1290/",
          "https://paatos-helsinki.6aika.fi/v1/event/1305/",
          "https://paatos-helsinki.6aika.fi/v1/event/1337/",
          "https://paatos-helsinki.6aika.fi/v1/event/1376/",
          "https://paatos-helsinki.6aika.fi/v1/event/1378/",
          "https://paatos-helsinki.6aika.fi/v1/event/1403/",
          "https://paatos-helsinki.6aika.fi/v1/event/1462/",
          "https://paatos-helsinki.6aika.fi/v1/event/1463/",
          "https://paatos-helsinki.6aika.fi/v1/event/1464/",
          "https://paatos-helsinki.6aika.fi/v1/event/1465/",
          "https://paatos-helsinki.6aika.fi/v1/event/1466/",
          "https://paatos-helsinki.6aika.fi/v1/event/1602/",
          "https://paatos-helsinki.6aika.fi/v1/event/1620/",
          "https://paatos-helsinki.6aika.fi/v1/event/1697/",
          "https://paatos-helsinki.6aika.fi/v1/event/1720/",
          "https://paatos-helsinki.6aika.fi/v1/event/1727/",
          "https://paatos-helsinki.6aika.fi/v1/event/1743/",
          "https://paatos-helsinki.6aika.fi/v1/event/1759/",
          "https://paatos-helsinki.6aika.fi/v1/event/1787/",
          "https://paatos-helsinki.6aika.fi/v1/event/1823/",
          "https://paatos-helsinki.6aika.fi/v1/event/1828/",
          "https://paatos-helsinki.6aika.fi/v1/event/1832/",
          "https://paatos-helsinki.6aika.fi/v1/event/1850/",
          "https://paatos-helsinki.6aika.fi/v1/event/1880/",
          "https://paatos-helsinki.6aika.fi/v1/event/1902/",
          "https://paatos-helsinki.6aika.fi/v1/event/1916/",
          "https://paatos-helsinki.6aika.fi/v1/event/1939/",
          "https://paatos-helsinki.6aika.fi/v1/event/1970/",
          "https://paatos-helsinki.6aika.fi/v1/event/1993/",
          "https://paatos-helsinki.6aika.fi/v1/event/2012/",
          "https://paatos-helsinki.6aika.fi/v1/event/2038/",
          "https://paatos-helsinki.6aika.fi/v1/event/2052/",
          "https://paatos-helsinki.6aika.fi/v1/event/2098/",
          "https://paatos-helsinki.6aika.fi/v1/event/2117/",
          "https://paatos-helsinki.6aika.fi/v1/event/2131/",
          "https://paatos-helsinki.6aika.fi/v1/event/2145/",
          "https://paatos-helsinki.6aika.fi/v1/event/2180/",
          "https://paatos-helsinki.6aika.fi/v1/event/2196/",
          "https://paatos-helsinki.6aika.fi/v1/event/2221/",
          "https://paatos-helsinki.6aika.fi/v1/event/2262/",
          "https://paatos-helsinki.6aika.fi/v1/event/2276/",
          "https://paatos-helsinki.6aika.fi/v1/event/2280/",
          "https://paatos-helsinki.6aika.fi/v1/event/2292/",
          "https://paatos-helsinki.6aika.fi/v1/event/2324/",
          "https://paatos-helsinki.6aika.fi/v1/event/2333/",
          "https://paatos-helsinki.6aika.fi/v1/event/2373/",
          "https://paatos-helsinki.6aika.fi/v1/event/2400/",
          "https://paatos-helsinki.6aika.fi/v1/event/2420/",
          "https://paatos-helsinki.6aika.fi/v1/event/2430/",
          "https://paatos-helsinki.6aika.fi/v1/event/2448/",
          "https://paatos-helsinki.6aika.fi/v1/event/2457/",
          "https://paatos-helsinki.6aika.fi/v1/event/2477/",
          "https://paatos-helsinki.6aika.fi/v1/event/2497/",
          "https://paatos-helsinki.6aika.fi/v1/event/2525/",
          "https://paatos-helsinki.6aika.fi/v1/event/2542/",
        ],
        "founding_date": null,
        "id": 19,
        "modified_at": "2017-05-05T06:12:49.984382Z",
        "name": "Kaupunginvaltuusto",
        "origin_id": "02900",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/19/",
      },
      Object {
        "classification": "Osasto",
        "created_at": "2017-05-05T06:12:49.992342Z",
        "data_source": "helsinki",
        "dissolution_date": "2013-10-31",
        "events": Array [],
        "founding_date": null,
        "id": 20,
        "modified_at": "2017-05-05T06:12:49.992380Z",
        "name": "Hallinto-osasto/Pel",
        "origin_id": "111100",
        "parent": null,
        "posts": Array [],
        "url": "https://paatos-helsinki.6aika.fi/v1/organization/20/",
      },
    ]
  }
>
  <Section
    pad={
      Object {
        "vertical": "medium",
      }
    }
  >
    <Box
      announce={false}
      className="grommetux-section"
      direction="column"
      focusable={true}
      pad={
        Object {
          "vertical": "medium",
        }
      }
      responsive={true}
      tag="section"
    >
      <section
        className="grommetux-box grommetux-box--direction-column grommetux-box--responsive grommetux-box--pad-vertical-medium grommetux-section"
        style={Object {}}
      >
        <Search
          align="left"
          iconAlign="end"
          inline={true}
          onDOMChange={[Function]}
          placeHolder="Hakusana"
          responsive={false}
          size="medium"
        >
          <div
            className="grommetux-search grommetux-search--icon-align-end grommetux-search--inline grommetux-search--medium"
          >
            <input
              autoComplete="off"
              className="grommetux-input grommetux-search__input"
              onChange={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onMouseUp={[Function]}
              placeholder="Hakusana"
              type="search"
            />
            <Search
              responsive={true}
            >
              <svg
                aria-label="search"
                className="grommetux-control-icon grommetux-control-icon-search grommetux-control-icon--responsive"
                height="24px"
                role="img"
                version="1.1"
                viewBox="0 0 24 24"
                width="24px"
              >
                <path
                  d="M15,15 L22,22 L15,15 Z M9.5,17 C13.6421356,17 17,13.6421356 17,9.5 C17,5.35786438 13.6421356,2 9.5,2 C5.35786438,2 2,5.35786438 2,9.5 C2,13.6421356 5.35786438,17 9.5,17 Z"
                  fill="none"
                  stroke="#000"
                  strokeWidth="2"
                />
              </svg>
            </Search>
          </div>
        </Search>
      </section>
    </Box>
  </Section>
</OrganizationSearch>
`;
